### PR TITLE
fix: add scroll to session list tab in session tab

### PR DESCRIPTION
**Pull request by ggstone0523**
Resolve: lablup/backend.ai-webui #1198

- This bug is caused by inner block in 'session tab' h3 block is overflowed when window size is narrow.
- Add scroll to session list tab in session tab to prevent overflow

**test image**
PR3

### CODE REVIEW

File: src/components/backend-ai-session-view.ts [hunk 1]
```diff
@@ -442,14 +446,17 @@ export default class BackendAiSessionView extends BackendAIPage {
...
           <h3 class="tab horizontal center layout" style="margin-top:0;margin-bottom:0;">
-            <div class="horizontal layout flex start-justified">
-            <mwc-tab-bar>
-              <mwc-tab title="running" label="${_t('session.Running')}" @click="${(e) => this._showTab(e.target)}"></mwc-tab>
-              <mwc-tab title="interactive" label="${_t('session.Interactive')}" @click="${(e) => this._showTab(e.target)}"></mwc-tab>
-              <mwc-tab title="batch" label="${_t('session.Batch')}" @click="${(e) => this._showTab(e.target)}"></mwc-tab>
-              <mwc-tab title="finished" label="${_t('session.Finished')}" @click="${(e) => this._showTab(e.target)}"></mwc-tab>
-              <mwc-tab title="others" label="${_t('session.Others')}" @click="${(e) => this._showTab(e.target)}"></mwc-tab>
-            </mwc-tab-bar>
+            <div class="scroll hide_scrollbar">
```
Comment (agatha197) [inline src/components/backend-ai-session-view.ts]: We use kebab-case in class. Please rename `hide_scrollbar` to `hide-scrollbar`.

**Review by Sujin-Kim1 — CHANGES_REQUESTED** [review]

**Comment by ggstone0523** [comment]
### The revision was completed by reflecting the comments.

1. change 'hide_scrollbar' to 'hide-scrollbar'

**Review by Sujin-Kim1 — APPROVED** [review]
LGTM.

File: src/components/backend-ai-session-view.ts [hunk 2]
```diff
@@ -464,9 +471,8 @@ export default class BackendAiSessionView extends BackendAIPage {
                   </mwc-menu>
                 </div>
               ` : html``}
-            </div>
             <div class="horizontal layout flex end-justified" style="margin-right:20px;">
-            <backend-ai-session-launcher location="session" id="session-launcher" ?active="${this.active === true}" isSupportingFab></backend-ai-session-launcher>
+              <backend-ai-session-launcher location="session" id="session-launcher" ?active="${this.active === true}" isSupportingFab></backend-ai-session-launcher>
             </div>
```
Comment (lizable) [inline src/components/backend-ai-session-view.ts:476]: @ggstone0523 Let's remove `isSupportingFab` attribute in Line 475. It seems like a legacy code and we are no longer using that attribute anymore.

**Review by lizable — APPROVED** [review]
I removed `isSupportingFab` attribute since we don't use it anymore.
Except that, it works fine.

**Comment by ggstone0523** [comment]
@lizable thanks!

**Review by inureyes — APPROVED** [review]
LGTM

**Pull request merged**